packages/api/lambdas/sqs-message-consumer.js

```diff
@@ -18,66 +18,95 @@ const Rule = require('../models/rules');
  * messages from SQS queue are processed
  */
 async function processQueues(event, dispatchFn) {
-  const model = new Rule();
-  const rules = await model.getRulesByTypeAndState('sqs', 'ENABLED');
+  const rulesModel = new Rule();
+  const rules = await rulesModel.queryRules({
```

Comment:
The only downside to this change is that `rulesModel.queryRules` throws an error if it finds no rules. But if this Lambda should do nothing if it finds no rules. And if it throws an error, it should be logged so I'm not sure if this is a problem

Reply:
Can we just handle the error and write a log message like 'No SQS rules configured. Aborting.'?

Reply:
For sure